docs/INSTALL-devel.rst

```diff
@@ -0,0 +1,286 @@
+Dependencies: 
+=============
+There are a few things that HaaS expects the operating system to have::
+
+  yum install epel-release bridge-utils  gcc  httpd  ipmitool libvirt libxml2-devel  libxslt-devel  mod_wsgi \
+  net-tools python-pip python-psycopg2 python-virtinst python-virtualenv qemu-kvm telnet vconfig virt-install
+
+
+HaaS also expects there to be a database.  Currently, HaaS supports SQLite and PostgreSQL.  
```

Comment:
"HaaS requires a database server and currently supports only SQLite and PostgreSQL"

I paraphrased it.